pkg/gardenlet/controller/add.go

```diff
@@ -87,6 +88,13 @@ func AddToManager(
 		return fmt.Errorf("failed adding ControllerInstallation controller: %w", err)
 	}
 
+	if err := (&managedseed.Reconciler{
+		Config:     *cfg.Controllers.ManagedSeed,
+		ChartsPath: charts.Path,
```

Comment:
Remove this line and instead default it in `AddToManager` like in https://github.com/gardener/gardener/blob/c11c86ae07d8ea784f5c41362cd41800f06bb3ed/pkg/gardenlet/controller/seed/seed/add.go#L49-L51